src/ui/components/Passport/Passport.tsx

```diff
@@ -0,0 +1,221 @@
+import { useCallback, useEffect, useRef, useState } from "react";
+
+import Icon from "replay-next/components/Icon";
+import hooks from "ui/hooks";
+import {
+  shouldShowAddComment,
+  shouldShowAddUnicornBadge,
+  shouldShowBreakpointEdit,
+  shouldShowConsoleNavigate,
+  shouldShowExploreSources,
+  shouldShowInspectElement,
+  shouldShowJumpToCode,
+  shouldShowJumpToEvent,
+  shouldShowLaunchCommandPalette,
+  shouldShowRecordReplay,
+  shouldShowSearchSourceText,
+  shouldShowUseFocusMode,
+} from "ui/utils/onboarding";
+
+import styles from "./Passport.module.css";
+
+const Passport: React.FC = () => {
```

Comment:
:speech_balloon: Note for future reference: stylistically we normally avoid using `React.FC`.  It's not necessary most of the time.

Reply:
Yeah I was curious about that. Thanks for the context, will change!